lib/msal-browser/src/app/ClientApplication.ts

```diff
@@ -151,7 +151,8 @@ export abstract class ClientApplication {
         const nativeCacheOptions: Required<CacheOptions> = {
             cacheLocation: BrowserCacheLocation.MemoryStorage,
             storeAuthStateInCookie: false,
-            secureCookies: false
+            secureCookies: false,
+            cacheMigrationEnabled: false
```

Comment:
There is another API called `migrateCacheEntries` in `BrowserCacheManager`. How do we distinguish these cases? Or does this boolean always translate to whatever the latest cache look up is? I am fine with the name if we stick to the later logic.

Reply:
This boolean controls all cache migrations, including the ADAL to MSAL one you mentioned. We don't need to distinguish the cases. Essentially, if you have recently upgraded to a new version of MSAL or migrated from ADAL to MSAL you should leave this flag on (Default). Then once your application has been using the latest version of MSAL long enough that most users' caches have been migrated you can feel free to turn this off and skip the migration step.